ui/css/base-styles.scss

```diff
@@ -31,7 +31,7 @@ html {
   */
   @media (prefers-color-scheme: dark) {
     &:not([data-theme]) {
-      color: var(--brand-colors-white-white000);
+      color: var(--brand-colors-white);
```

Comment:
This seems like the default font color for prefers color scheme dark. Check before/after screenshot of description as settings for that recording are set to system